source/administration/announcement-banner.rst

```diff
@@ -1,9 +1,9 @@
 Announcement Banner (E10)
 =========================
 
-Available in `Enterprise Edition E10 and higher <https://mattermost.com/pricing-self-managed/>`__.
+*Available in `Enterprise Edition E10 and higher <https://mattermost.com/pricing-self-managed/>`__.*
```

Comment:
The links is now broken. @justinegeffen

Reply:
Thanks, @amyblais, I'll rectify in a new PR.